sdk/e2e/11_deploy_router_upi_traffic_split_test.py

```diff
@@ -64,7 +64,7 @@ def test_deploy_router_upi_traffic_split():
                     field_source=FieldSource.PREDICTION_CONTEXT,
                     field="client_id",
                     operator="in",
-                    values=["1"],
+                    values=["7"],
```

Comment:
1 would return control, which is the default route and doesn't test traffic rules